addons/website_event_track_quiz/security/ir.model.access.csv

```diff
@@ -1,7 +1,7 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-event_quiz_access_event_user,event.quiz.access.event.user,model_event_quiz,event.group_event_user,1,0,0,0
-event_quiz_access_event_manager,event.quiz.access.event.manager,model_event_quiz,event.group_event_manager,1,1,1,1
-event_quiz_question_access_event_user,event.quiz.question.access.event.user,model_event_quiz_question,event.group_event_user,1,0,0,0
-event_quiz_question_access_event_manager,event.quiz.question.access.event.manager,model_event_quiz_question,event.group_event_manager,1,1,1,1
-event_quiz_answer_access_event_user,event.quiz.answer.event.user,model_event_quiz_answer,event.group_event_user,1,0,0,0
-event_quiz_answer_access_event_manager,event.quiz.answer.event.manager,model_event_quiz_answer,event.group_event_manager,1,1,1,1
+event_quiz_access_event_registration,event.quiz.access.event.registration,model_event_quiz,event.group_event_registration_desk,1,0,0,0
+event_quiz_access_event_user,event.quiz.access.event.user,model_event_quiz,event.group_event_user,1,1,1,1
+event_quiz_question_access_event_registration,event.quiz.question.access.event.registration,model_event_quiz_question,event.group_event_registration_desk,1,0,0,0
+event_quiz_question_access_event_user,event.quiz.question.access.event.user,model_event_quiz_question,event.group_event_user,1,1,1,1
+event_quiz_answer_access_event_registration,event.quiz.answer.event.registration,model_event_quiz_answer,event.group_event_registration_desk,1,0,0,0
+event_quiz_answer_access_event_user,event.quiz.answer.event.user,model_event_quiz_answer,event.group_event_user,1,1,1,1
```

Comment:
Well all this doesn't make too much sense.
At this point we might as well put the appropriate group on the quiz_id field on event.track so it's hidden on the form and we don't need to allow read access to all sub-models to the registration desk group.
I don't see why they would ever need it.

And arguably being able to see quiz answers is a bit of a ""private information"" even though it's not super important.